packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -216,9 +214,11 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
     if (isStickyTop || isStickyBottom) {
       let height = 0,
         width = 0;
-      if (this.nonStickyContent) {
+      if (this.nonStickyContent && this.nonStickyContent.firstElementChild) {
         height = this.nonStickyContent.offsetHeight;
-        width = this.nonStickyContent.scrollWidth;
+        width =
+          this.nonStickyContent.firstElementChild.scrollWidth +
```

Comment:
Please add a code comment on why this is done, and what are the assumptions.

Reply:
sure